src/site/includes/how-do-you-rate.drupal.liquid

```diff
@@ -1,5 +1,5 @@
 {% if false %}
```

Comment:
Do we still need this template?

Reply:
During certain edge cases of the build (off hours) sometimes experimental builds are executed where `false` and `true` are reversed. In those cases I think we are counting on this component rendering

Reply:
Just kidding - One day when we have a Medallia API rolled out I think we will at last get this one in use

Reply:
Oh wow hahahaha that scared me so much for a sec 😂 I was like "why in the..." lol